### PR TITLE
Historic

### PR DESCRIPTION
**Pull request by dderancourt**
creation of a component historic that display historic information, add a bit of style

### CODE REVIEW

**Review by laurentmahieu — CHANGES_REQUESTED** [review]
not on the master !!!!
little padawan

File: frontend/src/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import { BrowserRouter } from "react-router-dom";
+
```
Comment (laurentmahieu) [inline frontend/src/index.js:7]: why add a space?

File: frontend/src/components/style/Historic.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+$gradientColor1: #00aac6;
+$gradientColor2: #00c1b7;
+$gradientColor3: #00dc9f;
+$gradientColor4: #ffffff;
+$gradientColor5: #72bac52a;
+$gradientColor6: #00abc62a;
+
+.HistoricBox {
+  display: flex;
+  .ScrollBar {
```
Comment (laurentmahieu) [inline frontend/src/components/style/Historic.scss:10]: why is it empty ? normal ?

**Review by laurentmahieu — APPROVED** [review]

File: frontend/src/index.js (hunk 1 above)
Comment (dderancourt) [reply frontend/src/index.js:7]: i prefer add some space between redux import and other imports to add readability

File: frontend/src/components/style/Historic.scss (hunk 2 above)
Comment (dderancourt) [reply frontend/src/components/style/Historic.scss:10]: the scrollbar will be stylized in the future

**Review by LoicBrassart — APPROVED** [review]
👌

**Pull request merged**